jaeger-core/src/main/java/com/uber/jaeger/propagation/TextMapCodec.java

```diff
@@ -75,7 +76,7 @@ public SpanContext extract(TextMap carrier) {
     String debugId = null;
     for (Map.Entry<String, String> entry : carrier) {
       // TODO there should be no lower-case here
-      String key = entry.getKey().toLowerCase();
+      String key = entry.getKey().toLowerCase(Locale.ROOT);
```

Comment:
-1 : I don't know that root even means here, but the no-arg toLowerCase () method clearly says "using the default locale"

Reply:
Can you read the whole javadoc for this method and not only the title sentence?
https://docs.oracle.com/javase/8/docs/api/java/lang/String.html#toLowerCase--
It gives example of what is wrong and how one should use toLowerCase without surprises.

Reply:
fair enough, +1